### DISCUSSION TITLE
RNFirebase not working with react-redux-firebase and redux-thunk

### DISCUSSION THREAD

**Discussion by nirmalhk7** [invertase/react-native-firebase#5507] [Q&A]
I'm trying to use RNFirebase with react-redux-firebase and redux-thunk. Here's my `App.js`:
```
    <Provider store={store}>
      <ReactReduxFirebaseProvider {...rrfProps}>
        <NavigationContainer>
          <Stack.Navigator headerMode={'none'}>
            <Stack.Screen name="HomeNavigator" component={HomeNavigator} />
            <Stack.Screen name="Login" component={LoginScreen} />
            <Stack.Screen name="Sign Up" component={LoginScreen} />
          </Stack.Navigator>
        </NavigationContainer>
      </ReactReduxFirebaseProvider>
    </Provider>
```
Here are `store` and `rrfProps`:
```
export const store = createStore(
  rootReducer,

  applyMiddleware(thunk.withExtraArgument({getFirebase, getFirestore})),
  // compose(reactReduxFirebase(firebase, reduxFirebaseConfig))
);
export const rrfProps = {
  firebase: RNFirebase,
  config: {
    userProfile: 'users',
    useFirestoreForProfile: true,
  },
  dispatch: store.dispatch,
};
```
The `google-services.json` file is already in `android/app/src`. But when I try to fetch any of my Firestore collections this way:
```
const mapStateToProps = (state) => {
  return {
    offers: state,
  };
};
const mapDispatchToProps = (dispatch) => {
  return {
    add_update_offer: (offer, env) => dispatch(redux.actions.admin.offers.create(offer, env)),
  };
};
export default compose(
  connect(mapStateToProps),
  firestoreConnect([
    {
      collection: "offers",
      where: [["db", "==", "prod"]],
    },
  ])
)(ViewOffers);
```

I keep getting no data. Nothing. I dont even see the query I'm making either, everything in the `offers` prop is blank. Please help, Thanks!

**Comment by mikehardy** [comment]
I suggest stackoverflow as a better place to request help here, I don't use redux personally and I'm not sure you'll get many answers.
Strongly suggest keeping it to a single JS file (App.js) that can be dropped into the result of https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh so anyone that hopes to help can actually run the example https://duckduckgo.com/?t=canonical&q=minimum+reproducible+example
